src/utils/validator.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * 배열에 constructor의 instance가 아닌 값이 존재하는지 판별합니다.
+ * @param {unknown[]} instances 체크할 배열입니다.
+ * @param {Function} constructor 체크할 클래스입니다.
+ * @returns {boolean} instance가 아닌 값의 존재여부입니다.
+ */
+export const invalidInstanceElement = (instances, constructor) => {
+  const result = instances.some((instance) => !(instance instanceof constructor));
+  return result;
+};
```

Comment:
```suggestion
export const invalidInstanceElement = (instances, constructor) => 
  instances.some((instance) => !(instance instanceof constructor));
```

p4 - 이런식으로 줄여쓸 수 있을 것 같아요!